core/src/main/java/org/dcsa/conformance/core/AbstractComponentFactory.java

```diff
@@ -210,9 +210,9 @@ protected static String _sandboxIdPrefix(
       String standardName, String standardVersion, String scenarioSuite) {
     return "%s-%s-%s"
         .formatted(
-            standardName.replaceAll(" ", ""),
-            standardVersion.replaceAll("\\.", "").replaceAll("-", ""),
-            scenarioSuite.replaceAll(" ", "-"))
+            standardName.replace(" ", ""),
+            standardVersion.replaceAll("\\.", "").replace("-", ""),
+            scenarioSuite.replace(" ", "-"))
```

Comment:
`replace` also works on all occurrences. See e.g. https://medium.com/@KisinaDenis/what-is-the-difference-between-replace-vs-replaceall-method-in-java-string-class-6ca0e757ee5b

Reply:
Yes indeed. Will do that one too.

Reply:
Directly addressed more `replaceAll` in the code base, without using regexp.